components/auth0.key.manager/src/main/java/org/wso2/auth0/client/Auth0ConnectorConfiguration.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.auth0.client;
+
+import org.osgi.service.component.annotations.Component;
+import org.wso2.carbon.apimgt.api.model.ConfigurationDto;
+import org.wso2.carbon.apimgt.api.model.KeyManagerConnectorConfiguration;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+@Component(
+        name = "auth0.configuration.component",
+        immediate = true,
+        service = KeyManagerConnectorConfiguration.class
+)
+public class Auth0ConnectorConfiguration implements KeyManagerConnectorConfiguration {
+    @Override
+    public String getImplementation() {
+        return Auth0OAuthClient.class.getName();
+    }
+
+    @Override
+    public String getJWTValidator() {
+        return null;
+    }
+
+    @Override
+    public List<ConfigurationDto> getConnectionConfigurations() {
+        List<ConfigurationDto> configurationDtoList = new ArrayList<>();
+        configurationDtoList
+                .add(new ConfigurationDto(Auth0Constants.CLIENT_ID, "Client ID", "input",
+                        "Client ID of service Application", "",
```

Comment:
This should be either 'Client ID of Service Application' or 'Client ID of service application'

Reply:
fixed